docs/conf.py

```diff
@@ -36,16 +35,27 @@
 author = f"{project} project contributors"
 copyright = author  # pylint:disable=redefined-builtin
 
-# fmt: off
-# The short X.Y version
-version = ".".join(
-    get_scm_version(
-        local_scheme="no-local-version",
-    ).split(".")[:3],
-)
 
-# The full version, including alpha/beta/rc tags
-release = get_scm_version()
+# on_rtd is whether we are on readthedocs.org, this line of code grabbed from docs.readthedocs.org
+# https://docs.readthedocs.io/en/latest/faq.html?highlight=environ#how-do-i-change-behavior-for-read-the-docs
+on_rtd = os.environ.get("READTHEDOCS", None) == "True"
+if on_rtd:
+    rtd_version = os.environ["READTHEDOCS_VERSION"]
+    release = rtd_version
```

Comment:
Looks like one of the vars needs to be converted to a short version while the other should preserve the full one. The problem with this is that RTD does not know about those local and dev version details that setuptools-scm extracts from Git. So it will probably make it impossible to replicate the behavior exactly.

Reply:
I think the commit hash is somehow exposed on RTD but it's still not the same — the version base that is supposed to be calculated from Git for intermediate (in-between-the-tags) commits is nowhere to be found.